docs/estimators-fep.rst

```diff
@@ -16,8 +16,8 @@ and parse the datafiles separately for each alchemical leg using :func:`alchemly
     >>> from alchemlyb.parsing.gmx import extract_u_nk
     >>> import pandas as pd
 
-    >>> u_nk_coul = pd.concat([extract_u_nk(xvg, T=300) for xvg in bz['Coulomb']])
-    >>> u_nk_vdw = pd.concat([extract_u_nk(xvg, T=300) for xvg in bz['VDW']])
+    >>> u_nk_coul = alchemlyb.concat([extract_u_nk(xvg, T=300) for xvg in bz['Coulomb']])
+    >>> u_nk_vdw = alchemlyb.concat([extract_u_nk(xvg, T=300) for xvg in bz['VDW']])
```

Comment:
I assume you did a `git grep concat` and looked at every occurrence in the code?

Reply:
Yes.